run_action.py

```diff
@@ -174,28 +174,63 @@ def main():
 
     github_token = os.environ.get("INPUT_GITHUB_TOKEN")
     github_api_url = os.environ.get("GITHUB_API_URL")
+    
     request_url = "%s/repos/%s/issues/%s/comments" % (
         github_api_url,
         repo,
         args.pull_request_id,
     )
 
+    headers = {
+        "Authorization": "token %s" % github_token,
+    }
+    report = {
+        "body": message
+    }
+
+    one_comment = os.environ.get("INPUT_ONE_COMMENT")
+    is_update = False
+    if one_comment.lower() in ("true", "1"):
+        ## Search comments
+        pr_comments = requests.get(request_url,
+                                   headers=headers
+                                   ).json()
+                                
+        header_message = "## 📌 Duplicate code detection tool report\n"
+        for pr_comment in pr_comments[::-1]:
```

Comment:
> `pr_comments[::-1]`

Is there a benefit doing this instead of `for pr_comment in pr_comments:`?

Reply:
If there are several comments, then update the last

Reply:
Aha I see, makes sense but the intent wasn't obvious. Perhaps:

```suggestion
        # If the bot has posted many comments, update the last one
        for pr_comment in pr_comments[::-1]:
```

Reply:
Hear is a link to the repository:
https://github.com/dzibukalexander/readingfiles

Reply:
Looks good!